features/promotion/receiving_discount_based_on_number_of_items_from_specific_taxon.feature

```diff
@@ -0,0 +1,55 @@
+@promotion
+Feature: Receiving discount based on number of products from specific taxon
+    In order to pay less while buying required number of goods from promoted taxon
+    As a Customer
+    I want to receive discount for my purchase
+
+    Background:
+        Given the store operates on a single channel in "France"
+        And the store classifies its products as "T-Shirts" and "Mugs"
+        And the store has a product "PHP T-Shirt" priced at "€100.00"
+        And it belongs to "T-Shirts"
+        And the store has a product "Symfony T-Shirt" priced at "€150.00"
+        And it belongs to "T-Shirts"
+        And the store has a product "PHP Mug" priced at "€20.00"
+        And it belongs to "Mugs"
+        And there is a promotion "T-Shirts promotion"
+
+    @ui
+    Scenario: Receiving discount on order while buying required number of products from promoted taxon
+        Given the promotion gives "€50.00" off if order contains 2 products classified as "T-Shirts"
+        When I add 3 products "PHP T-Shirt" to the cart
+        Then my cart total should be "€250.00"
+        And my discount should be "-€50.00"
+
+    @ui
+    Scenario: Receiving discount on order while buying equal with required number of products from promoted taxon
```

Comment:
`Scenario: Receiving a discount on order while buying exactly the required number of products from a promoted taxon` ?